versions/build.hera.ver

```diff
@@ -1,3 +1,4 @@
 export stack_intel_ver=2021.5.0
 export stack_impi_ver=2021.5.1
+export spack_env=gsi-addon-dev
```

Comment:
is it possible to put a variable here such as:
```bash
export spack_mod_path="/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/gsi-addon-dev/install/modulefiles/Core"
```
and then in the `module_base.hera.lua` have something like:
```lua
local spack_mod_path=os.getenv('spack_mod_path') or "None"
prepend_path("MODULEPATH", spack_mod_path)
```

Reply:
Yes, I think so -- it sounds a lot less ugly, too.  I'll give that a try.

Reply:
Fixed and tested on all research systems.